jsonstore/jsonstore.go

```diff
@@ -524,3 +583,73 @@ func (app *JSONStoreApplication) Commit() types.ResponseCommit {
 func (app *JSONStoreApplication) Query(reqQuery types.RequestQuery) (resQuery types.ResponseQuery) {
 	return
 }
+
+func (app *JSONStoreApplication) InitChain(req types.RequestInitChain) types.ResponseInitChain {
+	fmt.Println("calling InitChain")
+	validators := req.GetValidators()
+	var mintValidators []interface{}
+	if validators != nil {
+		for _, element := range validators {
+			validator := Validator{
+				ID:        bson.NewObjectId(),
```

Comment:
that's a bug. We cannot generate random value for each like that cos it will create different IDs on on every validator node.